contrib/gentoo/sys-apps/knxd/files/knxd.init

```diff
@@ -6,7 +6,7 @@
 
 pidfile="/run/knxd/knxd-@SLOT@.pid"
 command="/usr/bin/knxd"
-command_args="${KNXD_OPTS} --pid-file=${pidfile} -d${KNXD_LOGFILE}"
+command_args="--pid-file=${pidfile} -d ${KNXD_LOGFILE} ${KNXD_OPTS}"
```

Comment:
Apparently that's either a bug in the argp library or I don't know how to use that correctly.
Use `--daemon FILENAME`.
I will investigate further.